misc/python/materialize/buildkite_insights/step_durations/analysis.py

```diff
@@ -196,6 +196,7 @@ def main(
     )
 
     parser.add_argument("--pipeline", choices=MZ_PIPELINES, default="tests", type=str)
+    # TODO: Should also take --build-parallel-job
```

Comment:
Can you explain?

Reply:
I noticed that this does not work when a job is split into multiple shards. I left this TODO for you (or whoever wants to  analyze such steps with buildkite_insights next)

Reply:
Got it! I will have a look at it.